main.py

```diff
@@ -52,11 +56,19 @@ def get_auth_email_token() -> Tuple[str, str]:
         token = os.environ["KTAPI_AUTH_TOKEN"]
         return email, token
     except KeyError:
-        print("You have to specify KTAPI_AUTH_EMAIL and KTAPI_AUTH_TOKEN first")
+        logger.error("You have to specify KTAPI_AUTH_EMAIL and KTAPI_AUTH_TOKEN environment variables first")
         sys.exit(1)
 
 
+# Run production server: gunicorn --workers=1 'main:run()'
+def run() -> flask.Flask:
+    email, token = get_auth_email_token()
```

Comment:
I think that it should be next to `config = ConfigYAML("config.yaml")` in WebApp.init(), because reading configuration from env has high cohesion with reading configuration from file.